test/framework/src/test/java/org/elasticsearch/test/test/LoggingListenerTests.java

```diff
@@ -33,6 +33,13 @@
 import static org.hamcrest.CoreMatchers.nullValue;
 
 public class LoggingListenerTests extends ESTestCase {
+
+    public void testTestRunStartedSupportsClassInDefaultPackage() throws Exception {
+        LoggingListener loggingListener = new LoggingListener();
+        Description description = Description.createTestDescription(Class.forName("Dummy"), "dummy");
+        loggingListener.testRunStarted(description);
```

Comment:
Thanks for updating. Would you mind pushing a comment explaining that this would throw an exception without your fix?